androidApp/src/main/java/com/example/tasky/android/agenda/screen/AgendaDetailsScreen.kt

```diff
@@ -146,7 +149,16 @@ sealed interface AgendaDetailsScreenEvent {
     ) : AgendaDetailsScreenEvent
 
     data class OnStartDateChange(
-        val newDate: Long,
+        val newDate: LocalDateTime,
+    ) : AgendaDetailsScreenEvent
+
+    data class OnEndTimeChange(
+        val newHour: Int,
+        val newMinute: Int,
+    ) : AgendaDetailsScreenEvent
+
+    data class OnEndDateChange(
+        val newDate: LocalDateTime,
     ) : AgendaDetailsScreenEvent
```

Comment:
If just a date changes, could it make more sense to work with `LocalDate` instead of `LocalDateTime`?